babel.config.js

```diff
@@ -53,6 +53,18 @@ module.exports = function(api) {
           useBuiltIns: true
         }
       ],
+      [
```

Comment:
I don't know, I didn't add it myself. I think it got added when I added a `private` method in `posts_controller.rb`. The private method is called `post_params` and I was just following the make-a-blog [instructions video](https://www.youtube.com/watch?v=BI_VnnOLSKY). I'm guessing it got added because of that? What does `babel.config.js` do?